components/director/run.sh

```diff
@@ -13,6 +13,7 @@ ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 SKIP_DB_CLEANUP=false
 REUSE_DB=false
+DISABLE_ASYNC_MODE=true
```

Comment:
It is a personal opinion, but I do not like reversed logic variables. 
I prefer EnabledX vs DisabledX, as boolean values are easily guessed later in code:
TRUE=ENABLED
FALSE=DISABLED

Reply:
The default behavior is with async enabled, so explicitly disabling it seemed more reasonable - it will be harder to misconfigure it